src/open_inwoner/js/components/form/LoginForm.js

```diff
@@ -50,34 +50,38 @@ export class LoginFormFocus {
 
   activateTabFromHash() {
     const hash = window.location.hash
-    const particulierLink = document.querySelector(
-      '.tab__header[href="/accounts/login/#particulier"]'
-    )
-    const zakelijkLink = document.querySelector(
-      '.tab__header[href="/accounts/login/#zakelijk"]'
-    )
-    const particulierTab = document.getElementById('particulier')
-    const zakelijkTab = document.getElementById('zakelijk')
+    const particulierPanel = document.getElementById('particulier_panel')
+    const zakelijkPanel = document.getElementById('zakelijk_panel')
 
     if (hash.includes('zakelijk')) {
-      particulierLink.classList.remove('active')
+      const zakelijkTab = document.getElementById('zakelijk_tab')
+      const particulierTab = document.getElementById('particulier_tab')
+
       particulierTab.classList.remove('active')
-      particulierTab.classList.add('hide')
+      particulierPanel.classList.remove('active')
+      particulierPanel.classList.add('hide')
 
-      zakelijkTab.classList.remove('hide')
+      zakelijkPanel.classList.remove('hide')
+      zakelijkPanel.classList.add('active')
       zakelijkTab.classList.add('active')
-      zakelijkLink.classList.add('active')
     } else {
-      particulierTab.classList.remove('hide')
-      particulierLink.classList.add('active')
-      particulierTab.classList.remove('active')
+      const zakelijkHeader = document.getElementById('zakelijk_tab')
+      const particulierHeader = document.getElementById('particulier_tab')
```

Comment:
In de if heb je dit:

    const zakelijkTab = document.getElementById('zakelijk_tab')
    const particulierTab = document.getElementById('particulier_tab')

Maar in de else heten ze anders :|

      const zakelijkHeader = document.getElementById('zakelijk_tab')
      const particulierHeader = document.getElementById('particulier_tab')

Je kan die twee regels ook gewoon boven de if zetten.

Reply:
Ik was tegelijkertijd teveel dingen aan het hernoemen waardoor Find-and-replace niet vlekkeloos ging :-D bij deze gecorrigeerd.